index.js

```diff
@@ -57,10 +60,27 @@ module.exports = function (options) {
 		}
 
 		try {
-			var errors = checker.checkString(file.contents.toString(), file.relative);
+			var fixResults;
+			var errors;
+			var changed = false;
```

Comment:
Your change detection is also optimizing for the wrong thing. Writing to disk is super fast compared to actually fixing the string (needs to parse, iterate, and stringify the AST), which makes your check a lot slower than just using `gulp-changed`.